### DISCUSSION TITLE
Update to deploy.yml

### DISCUSSION THREAD

**Discussion by mtello22** [gethyas/hyas#481] [Ideas]
First of all, thank you for creating this awesome framework for personal sites. 
I would like to contribute my two cents on the issues I ran as a new user:
When deploying my code to GitHub Pages, I had some errors because GitHub couldn't find Hugo. So I added the following steps before the Checkout step:
`      - name: Install Hugo CLI
        run: |
          wget -O ${{ runner.temp }}/hugo.deb https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_linux-amd64.deb \
          && sudo dpkg -i ${{ runner.temp }}/hugo.deb          
      - name: Install Dart Sass
        run: sudo snap install dart-sass`
I took the code from the base [hugo deployment](https://gohugo.io/hosting-and-deployment/hosting-on-github/), I don't know if other new users may run into the same issue.

**Comment by h-enk** [comment]
I'm updating the documentation for Hyas 2.4 (and Doks 1.6) — your input will find its place. Thanks!

**Reply by h-enk** [reply]
[8072fab8f79f36bea0c16be01d32f5a6e8f4cd4a](https://github.com/gethyas/docs/commit/8072fab8f79f36bea0c16be01d32f5a6e8f4cd4a)
